bddl/activity_definitions/set_up_a_mouse_cage/problem0.bddl

```diff
@@ -30,7 +30,7 @@
     (:goal 
         (and 
             (filled ?bowl.n.01_1 ?petfood.n.01_1) 
-            (overlaid ?tissue.n.02_1 ?hutch.n.01_1) 
+            (ontop ?tissue.n.02_1 ?hutch.n.01_1)
```

Comment:
`tissue.n.02` is a non-leaf class that includes a roll of tissue paper, so imo it's not very suitable. `hand_towel.n.01` (which is a deformable and can be overlaid) would be better - that's what I did on `develop`.

Reply:
gotcha - let me update that here too then to mirror that change

Reply:
done!